Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -4915,11 +4920,18 @@ private void ProcessFileChanges(LoggingContext loggingContext, SchedulerState sc
         {
             InputChangeList inputChangeList = null;
 
-            if (m_configuration.Schedule.InputChanges.IsValid)
+            var inputChangesFilePath = Environment.GetEnvironmentVariable("[Sdk.BuildXL]inputChanges");

```

Comment:
>var inputChangesFilePath = Environment.GetEnvironmentVariable("[Sdk.BuildXL]inputChanges"); [](start = 12, length = 91)

The ability to consume an externally specified changes is something that's potentially useful to BuildXL beyond this code coverage feature. I don't really want to couple it to the QTest sdk. It's also a bit sneaky to toggle the BuildXL behavior just based on an environment variable. I think we should preserve the command line configuration.